src/superphot_plus/format_data_ztf.py

```diff
@@ -10,11 +13,26 @@
 
 
 def import_labels_only(input_csvs, allowed_types, fits_dir=None, redshift=False):
-    """
-    Import all features and labels, convert to label and features
+    """Import all features and labels, convert to label and features
```

Comment:
Original docstring was accidentally copied over from import_features_and_labels. This function only imports the ZTF names and class labels (what type of SN it is), not the input features

Reply:
How do these sound as descriptions?

**import_labels_only**: Filters CSVs for rows where label is in allowed_types and returns
    names, labels, and optionally redshift numpy arrays.

**import_features_and_labels**: Filters CSVs for rows where label is in allowed_types and returns 
    names, labels, and features.